ShaderTool/Command/Map.cs

```diff
@@ -187,6 +187,20 @@ class Map {
             resourceStream.Write(BitConverter.GetBytes(TGR_VERSION));
 
             // TODO
+            MapData mapData = Load(mapName);
+            string[] mapTextureName = mapData.textureNames;
+            for (int i = 0; i < mapTextureName.Length; i++)
+            {
+                string textureFilePath = Program.ResourcesFolder + "\\" + mapTextureName[i] + ".tex";
+                if (!File.Exists(textureFilePath))
+                {
+                    Console.WriteLine("{0} is not a texture.", mapTextureName);
```

Comment:
```suggestion
                    Console.WriteLine("{0} is not a texture.", mapTextureName[i]);
```
You don't want to print out the whole array just the name you are currently trying to read so in this case you need the index